doc/user_doc/vic_installation/vic_installation_prereqs.md

```diff
@@ -0,0 +1,44 @@
+# Environment Prerequisites for vSphere Integrated Containers Installation
+
+Before you install vSphere Integrated Containers, you must ensure that your infrastructure meets certain requirements.
+
+## Supported Installation Environments
+
+The current builds of vSphere Integrated Containers only run on Linux OS systems. The vSphere Integrated Containers repository includes a preconfigured  Vagrant Ubuntu VM, DevBox, in which you can build the binaries and run the `vic-machine` utility to deploy virtual container hosts. For information about how to set up DevBox, see [Use the Vagrant DevBox VM to Build the vSphere Integrated Containers Binaries](set_up_devbox.md).
+
+
+## Supported vSphere Configurations
+
+You can install vSphere Integrated Containers in the following vSphere setups:
+
+* Standalone ESXi 6.0 or 5.5 host that is not managed by a vCenter Server instance.
```

Comment:
5.5 is currently planned for 1.0. It should work but do we want to call that out?